src/test/java/seedu/address/logic/commands/ModelStubWithAddressBook.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.UniqueTagList;
+
+/**
+ * Model stub with AddressBook functionality
+ */
+class ModelStubWithAddressBook extends ModelStub {
+
+    private final AddressBookStubWithTag ab = new AddressBookStubWithTag();
+
+    @Override
+    public void addTag(Tag tag) {
+        ab.addTag(tag);
+    }
+
+    @Override
+    public ReadOnlyAddressBook getAddressBook() {
+        return ab;
+    }
+}
+
+class AddressBookStubWithTag implements ReadOnlyAddressBook {
```

Comment:
Maybe the AddressBook should be renamed to HelloFile? But I think it can be done later.